<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditConnectionsDlg</class>
 <widget class="QDialog" name="CEditConnectionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Connections</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Change to make</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="c_join_type">
          <property name="text">
           <string>&amp;Add connections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="c_radio2">
          <property name="text">
           <string>&amp;Merge connections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="c_radio3">
          <property name="text">
           <string>&amp;Break connections</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="c_pin_to_sel">
          <property name="text">
           <string>&amp;Selected area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="c_sel_to_pin">
          <property name="text">
           <string>&amp;Pinned area</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CEditConnectionsDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CEditConnectionsDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
